inspirehep/config.py

```diff
@@ -1496,6 +1504,45 @@ def _(x):
             "default_order": 'desc',
             "order": 2,
         },
+        "core": {
+            "title": 'Core (Desc)',
+            "fields": [
+                {
+                    '_extra_data.relevance_prediction.scores.CORE':
+                        {
+                            "ignore_unmapped": True,
+                            "order": "desc"
+                        }
+                }
+            ],
+            "order": 3,
+        },
+        "non_core": {
+            "title": 'Non Core (Desc)',
+            "fields": [
+                {
+                    '_extra_data.relevance_prediction.scores.Non-CORE':
+                        {
+                            "ignore_unmapped": True,
+                            "order": "desc"
+                        }
+                }
+            ],
+            "order": 4,
+        },
+        "rejected": {
+            "title": 'Rejected (Asc)',
+            "fields": [
+                {
+                    '_extra_data.relevance_prediction.scores.Rejected':
+                        {
+                            "ignore_unmapped": True,
+                            "order": "asc"
+                        }
+                }
+            ],
+            "order": 5,
+        }
```

Comment:
The problem here is this does not satisfy what the user expects to see. An article like https://arxiv.org/abs/hep-th/9711200, which is hugely core, has a huge CORE score, but also a big (in absolute value) Rejected score. If we implement this feature like this then all hugely core articles will come before all records that were actually rejected...